aleph/logic/entities.py

```diff
@@ -113,12 +118,76 @@ def entity_tags(entity, authz=None):
 
     res = _filters_faceted_query(facets, authz=authz)
     for (_, alias, field, _, value) in facets:
-        total = res.get(alias, 0)
+        total = res.get(alias, {}).get('count', 0)
         if total > 1:
             yield (field, value, total)
 
 
-def _filters_faceted_query(facets, authz=None):
+def entity_expand_nodes(entity, properties=None, include_entities=False, authz=None):  # noqa
+    proxy = model.get_proxy(entity)
+    schema = proxy.schema
+    facets = []
+    reversed_properties = []
+    literal_value_properties = []
+    matchable_prop_types = [
+        registry.name, registry.email, registry.identifier, registry.iban,
+        registry.phone, registry.address, registry.url, registry.checksum,
+    ]
```

Comment:
We have to many of these lists. Let's discuss this and come up with a clear plan.